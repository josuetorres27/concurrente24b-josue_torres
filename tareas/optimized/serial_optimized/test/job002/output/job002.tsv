plate000.bin	1.000000	50	1000.000000	0.0001	1	0000/00/00	00:00:01
plate001.bin	1.000000	50	1000.000000	1e-14	129570	0000/00/01	11:59:30
plate002.bin	1.000000	25	1000.000000	1e-14	250939	0000/00/02	21:42:19
plate003.bin	1.000000	50	1000.000000	0.0001	16566	0000/00/00	04:36:06
plate004.bin	1.000000	50	1000.000000	0.0001	20795	0000/00/00	05:46:35
plate005.bin	1.000000	50	1000.000000	0.0005	52982	0000/00/00	14:43:02
plate006.bin	60.000000	33.8	1200.000000	5e-09	5433	0000/00/03	18:33:00
plate007.bin	120.000000	10.7	1500.000000	5e-14	22761	0000/01/00	14:42:00
