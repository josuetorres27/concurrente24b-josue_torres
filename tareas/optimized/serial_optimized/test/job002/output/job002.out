plate000.bin	1	50	1000	0.0001	1	0000/00/00	00:00:01
plate001.bin	1	50	1000	1e-14	129570	0000/00/01	11:59:30
plate002.bin	1	25	1000	1e-14	250939	0000/00/02	21:42:19
plate003.bin	1	50	1000	0.0001	16566	0000/00/00	04:36:06
plate004.bin	1	50	1000	0.0001	20795	0000/00/00	05:46:35
plate005.bin	1	50	1000	0.0005	52982	0000/00/00	14:43:02
plate006.bin	60	33.8	1200	5e-09	5433	0000/00/03	18:33:00
plate007.bin	120	10.7	1500	5e-14	22761	0000/01/00	14:42:00
